src/qtiItem/core/Element.js

```diff
@@ -200,17 +200,17 @@ var Element = Class.extend({
         });
         return elts;
     },
-    getUsedClasses: function() {
+    getUsedClasses: function () {
         var ret = [this.qtiClass],
             composingElts = this.getComposingElements();
 
-        _.each(composingElts, function(elt) {
+        _.each(composingElts, function (elt) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
        _.forEach(composingElts, function (elt) {
```